client/flutter/lib/pages/travel_advice.dart

```diff
@@ -1,85 +1,132 @@
-              child: PageButton(
-                Color(0xff008DC9),
-                S.of(context).travelAdvicePageButtonGeneralRecommendations,
-                () => launch(S.of(context).travelAdvicePageButtonGeneralRecommendationsLink),
-                description:
-                    S.of(context).travelAdvicePageButtonGeneralRecommendationsDescription,
-                verticalPadding: 16,
-                titleStyle:
-                    TextStyle(fontSize: 16, fontWeight: FontWeight.bold),
-                descriptionColor: Colors.white,
-              ),
-            )
-          ])),
+          _adviceContent != null ? _buildBody(context) : _buildLoading(),
         ],
         title: S.of(context).homePagePageButtonTravelAdvice);
   }
+
+  SliverList _buildBody(BuildContext context) {
+    return SliverList(
+        delegate: SliverChildListDelegate([
+      Container(
+        color: Color(0xffD82037),
+        padding: EdgeInsets.symmetric(horizontal: 25, vertical: 10),
+        child: Html(
+          data: _adviceContent?.banner ?? "",
+          defaultTextStyle: TextStyle(
+            color: Colors.white,
+            // The Html widget isn't using the textScaleFactor so here's
+            // the next best place to incorporate it
+            fontSize: 18 * MediaQuery.textScaleFactorOf(context),
+            height: 1.2,
+          ),
+        ),
+      ),
+      Padding(
+        padding: const EdgeInsets.symmetric(horizontal: 16.0, vertical: 10.0),
+        child: Semantics(
+          header: true,
+          child: Text(
+            S.of(context).travelAdvicePageListTitle,
+            style: TextStyle(
+                fontSize: 24,
+                fontWeight: FontWeight.w800,
+                color: Color(0xff050C1D)),
+          ),
+        ),
+      ),
+      ..._getItems(context),
+      SizedBox(
+        height: 30,
+      ),
+      if (_adviceContent?.recommendationsLink != null) _buildLinkButton(context)
+    ]));
+  }
+
+  Padding _buildLinkButton(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 8.0, vertical: 10.0),
+      child: PageButton(
+        Color(0xff008DC9),
+        S.of(context).travelAdvicePageButtonGeneralRecommendations,
+        () => launch(_adviceContent.recommendationsLink),
+        description: _adviceContent?.recommendations ?? "",
+        verticalPadding: 16,
+        titleStyle: TextStyle(fontSize: 16, fontWeight: FontWeight.bold),
+        descriptionColor: Colors.white,
+      ),
+    );
+  }
+
+  SliverToBoxAdapter _buildLoading() {
+    return SliverToBoxAdapter(
+        child: Padding(
+      padding: const EdgeInsets.all(48.0),
+      child: CupertinoActivityIndicator(),
+    ));
+  }
+
+  List<Widget> _getItems(BuildContext context) {
+    return (_adviceContent?.items ?? []).map((item) {
+      return TravelAdviceListItem(
+          imageSrc: 'assets/travel_advice/${item.imageName}.png',
```

Comment:
File an issue to scrub all variables that go into path components.

They should have solely [a-zA-Z0-9/].

Reply:
#1028